v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner_Flex.md

```diff
@@ -278,21 +274,20 @@ resource "google_dataflow_flex_template_job" "sourcedb_to_spanner_flex" {
     projectId = "<projectId>"
     outputDirectory = "<outputDirectory>"
     # sourceDbDialect = "MYSQL"
-    # jdbcDriverJars = "gs://your-bucket/driver_jar1.jar,gs://your-bucket/driver_jar2.jar"
+    # jdbcDriverJars = ""
     # jdbcDriverClassName = "com.mysql.jdbc.Driver"
     # username = ""
     # password = ""
     # tables = ""
     # numPartitions = "0"
     # spannerHost = "https://batch-spanner.googleapis.com"
-    # maxConnections = "-1"
+    # maxConnections = "0"
```

Comment:
Generally `-1` is preferred (which is also the default). -1 indicates no limits on connections. The user can configure a non-negaitve value in case they need to reduce the number of active connections per worker.